### PR TITLE
Update openshift-ci base image to fedora 34

### PR DESCRIPTION
**Pull request by sabinaaledort**

### CODE REVIEW

File: openshift-ci/Dockerfile.tools [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/fedora/fedora:31-x86_64
+FROM quay.io/fedora/fedora:33-x86_64
```
Comment (yuvalk) [inline openshift-ci/Dockerfile.tools:1]: y not 34 which is the latest?
and contemplating whether we might want something more rigid and LTS (ie not fedora at all)

Comment (sabinaaledort) [reply openshift-ci/Dockerfile.tools:1]: i will update to 34.
maybe we can use centos? @SchSeba what do you think?

Comment (yuvalk) [reply openshift-ci/Dockerfile.tools]: centos moved to be a stream os
it is not suitable for CI
as you are getting something unpredictable :-(

Comment (SchSeba) [reply openshift-ci/Dockerfile.tools]: I am open to everything you prefer this is just a CI container

Comment (yuvalk) [reply openshift-ci/Dockerfile.tools]: debian ? :-D

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
flaky ovs_qos tests

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
flaky ovs_qos tests were removed
https://github.com/openshift-kni/cnf-features-deploy/pull/595

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
docker rate limit

**Comment by sabinaaledort** [comment]
/test e2e-gcp
added logs and artifacts to a followup story.

**Comment by sabinaaledort** [comment]
/test e2e-gcp
added logs and artifacts to a followup story.

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
/test e2e-gcp-ran-profile
/test e2e-gcp
/test images
docker rate limit

**Pull request merged**